### ISSUE TITLE
`rev` option: bundle hash is not stable

### ISSUE THREAD

**Issue by RenJing** [aurelia/cli#472]
**I'm submitting a bug report**

* **Library Version:**
0.24.0

**Please tell us about your environment:**
* **Operating System:**
Windows Server 2012

* **Node Version:**
6.1.0

* **NPM Version:**
3.9.0

**Current behavior:**
We enabled `rev` option for prod, and the bundle hash changes every time we run `npm run prod` even source files don't have any changes. We compare the bundles, and found the source files are not loaded in a stable order when generate bundle.

**Expected/desired behavior:**
* **What is the expected behavior?**
The bundle hash should not change if no changes in source files.

Steps to reproduce:

1. Define a bundle in aurelia.json like this: 
```
    {
        "name": "the-bundle.js",
        "source": [
          "[**/folder/**/*]"
        ]
    }
```
2. Create some components(.html and .js pairs) in `folder`. There are about 30 components in my case.
3. Enable rev for prod
4. Run `npm run prod` several times. 

* **What is the motivation / use case for changing the behavior?**

1. When we do a deployment, we only want to replace the bundle with changes.
2. Making the hash stable can help on website performance.

**Comment by EisenbergEffect** [comment]
@AStoker Is this a dup? Did we already have a fix merged for this but not yet released?

**Comment by sbolel** [comment]
@EisenbergEffect are https://github.com/aurelia/cli/issues/378 and https://github.com/aurelia/cli/pull/427 the ones you're thinking of perhaps?

**Comment by EisenbergEffect** [comment]
Yep. Thanks!

**Comment by RenJing** [comment]
Seems this issue is not fixed on latest version 0.26.1

**Comment by AStoker** [comment]
@RenJing, can you provide me with the details as to how this isn't working? Are you sure you've upgraded the  versions correctly, both the local version and your globally installed version?

**Comment by ghidello** [comment]
I just reproduced the issue with a fresh 0.27.0 install creating new default project, i.e.

> Name: es6
> Platform: Web
> Transpiler: Babel
> Markup Processor: Minimal Minification
> CSS Processor: None
> Unit Test Runner: Karma
> Editor: Visual Studio Code

After setting the rev option to true in aurelia.json, i.e.

> "options": {
>   "minify": "stage & prod",
>   "sourcemaps": "dev & stage",
>   **"rev": true**
> }

I executed 4 times the command `au build` and I found 4 different vendor bundles in the script folder. Comparing the files the differences appear to be related to the order of the aurelia dependencies included in the bundle.
